apps/frontend/src/libs/enums/notification-message.enum.ts

```diff
@@ -1,5 +1,6 @@
 const NotificationMessage = {
 	CONTRIBUTOR_MERGE_SUCCESS: "Contributor was successfully merged.",
+	CONTRIBUTOR_SPLIT_SUCCESS: "Contributors was successfully splitted.",
```

Comment:
```suggestion
	CONTRIBUTOR_SPLIT_SUCCESS: "Contributor was successfully splitted.",
```
or
```suggestion
	CONTRIBUTOR_SPLIT_SUCCESS: "Contributors were successfully splitted.",
```

I preffer first one, cause we are splitting one contributor